test/arrays/deitz/many/test_many_arrays_of_records.chpl

```diff
@@ -12,7 +12,9 @@ record R2 {
 
 for param i in 1..p {
   var A1: [1..2] R1(i);
-  writeln(A1);
+  writeln("(p = ", A1[1].p, ", x = ", A1[1].x,
+          ") (p = ", A1[2].p, ", x = ", A1[2].x, ")");
```

Comment:
Could this case be handled more elegantly (arguably) by adding a readWriteThis() (or whatever the preferred approach is) method to the R1 record that printed both p and x?